src/relay/transforms/annotate_target.cc

```diff
@@ -203,16 +242,18 @@ class AnnotateTargetRewriter : public ExprRewriter {
         }
       }
     }
-    supported_targets.push_back("default");  // Make default as the last option.
-
+    supported_targets.push_back(default_target);  // Make default as the last option.
+    // Visit and mutate arguments after the target of this op has been determined.
+    Call post_call = Downcast<Call>(post);
+    if (pre->op->IsInstance<VarNode>()) {
```

Comment:
Could you elaborate why a CallNode may have a VarNode as its op?

Reply:
The test case is tests/python/relay/test_pass_annotate_target.py::test_while_let